core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5005,6 +5007,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.underfs.gcs.directory.suffix";
     public static final String UNDERFS_GCS_OWNER_ID_TO_USERNAME_MAPPING =
         "alluxio.underfs.gcs.owner.id.to.username.mapping";
+    public static final String UNDERFS_HDFS_CONFIGURATION_PREFIX = "kv@";
```

Comment:
This is not a PropertyKey name, right? Maybe this should go to Constants.java.

Reply:
Yeah, done.